clients/extension/src/pages/popup/pages/main/index.tsx

```
@@ -40,10 +46,61 @@ const ConnectedApp = styled.div`
+    path: "m/44'/118'/0'/0/0"
+  }
+]
+
 export const MainPage = () => {
   const vault = useCurrentVault()
   const { colors } = useTheme()
   const navigate = useAppNavigate()
+  const walletCore = useWalletCore()
+
+  const derivedAddresses = useMemo(() => {
+    console.log("derivedAddresses")
```

ESLint

[error] Replace `"derivedAddresses"` with `'derivedAddresses'`

(prettier/prettier)